src/Apps/Legion/Components/CookSource/CookSource.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import CookSource from './CookSource';
+import { ICookSourceProps } from './CookSource.types';
+
+const wrapperStyle = { width: '400px', height: '600px', padding: 8 };
+
+export default {
+    title: 'Apps/Legion/Components/CookSource',
+    component: CookSource,
+    decorators: [getDefaultStoryDecorator<ICookSourceProps>(wrapperStyle)]
+};
+
+type CookSourceStory = ComponentStory<typeof CookSource>;
+
+const Template: CookSourceStory = (args) => {
```

Comment:
yes it works, it is a standalone component, and since it is a mock story the useADXAdapter hook internally will use MockAdapter to fetch the data since there is no context with adapter is provided.